servers/republik/server.js

```diff
@@ -111,7 +115,8 @@ const run = async (workerId, config) => {
       ...defaultContext,
       t,
       signInHooks,
-      mail,
+      mail: crowdfundingsMail,
+      auth,
```

Comment:
I'm not a fan of this, as it adds an inconstitency in the usage of the critical auth code. Searching the code for import of `require('@orbiting/backend-modules-auth')` doesn't work anymore. If you really want to get down that route you should at least include it in the `server/publikator/server.js` as well, so that it's available in all contextes. Actually you should include it in all context creations (tests).